src/lib/client/Client.ts

```diff
@@ -128,7 +141,7 @@ export const enum ClientEvents {
 	VoiceStateUpdate = 'voiceStateUpdate',
 	WebhooksUpdate = 'webhooksUpdate',
 	WSDebug = 'wsDebug',
-	WTF = 'wtf'
+	WTF = 'wtf',
```

Comment:
```suggestion
	WTF = 'wtf'
```
begone you foul comma!